deploy-iamlive-ecs.yaml

```diff
@@ -0,0 +1,226 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Run iamlive on ECS"
+
+Parameters:
+  VPC:
+    Type: AWS::EC2::VPC::Id
+  SubnetA:
+    Type: AWS::EC2::Subnet::Id
+  SubnetB:
+    Type: AWS::EC2::Subnet::Id
+  Image:
+    Type: String
+    Default: skuenzli/iamlive-docker:certs-vol
+  InitImage:
+    Type: String
+    Default: skuenzli/iamlive-docker:ecs-init-certs-vol
```

Comment:
Can use `unfor19/iamlive-docker` image repository if you have up to date tags for both the main iamlive and init image.